contributor-guides/astro-maintainers-guide-to-docs.md

```diff
@@ -139,6 +139,8 @@ Typically, this will be an approval from another maintainer, often the Docs Lead
 
 ### `Co-authored-by:` credit
 
+**NEW!** Use [Kevin's Co-author Generator](https://co-author-generator.netlify.app/) to automatically get a list of all contributors to a PR to copy and paste in the "Squash and Merge" message. To add any individual manually, follow the instructions below.
```

Comment:
```suggestion
**NEW!** Use [Kevin's Co-author Generator](https://coauthor.kevinzunigacuellar.com/) to automatically get a list of all contributors to a PR to copy and paste in the "Squash and Merge" message. To add any individual manually, follow the instructions below.
```

Reply:
Do I need to test this URL to make sure it doesn't, in fact, just make @kevinzunigacuellar a co-author on every PR??  :laughing: